### DISCUSSION TITLE
Please add "aspect ratio bucketing" if possible

### DISCUSSION THREAD

**Discussion by BigDogg99** [TheLastBen/fast-stable-diffusion#1177] [Ideas]
https://github.com/NovelAI/novelai-aspect-ratio-bucketing

with this we dont need to crop image for training

**Comment by nawnie** [comment]
actually with bucketing croping becomes alot more important especially if you up the batch size, still be nice to have

**Comment by JD1234JD1234** [comment]
is this possible yet? cant get kohya colab to work

**Comment by vangogh-coder** [comment]
Dear allİ am avaible for new mission.iPhone’umdan gönderildiJD1234JD1234 ***@***.***> şunları yazdı (6 Mar 2023 22:31):﻿
is this possible yet? cant get kohya colab to work
